src/components/PastMeetups.js

```diff
@@ -1,75 +1,92 @@
-import Image from 'gatsby-image'
-import { Box, Heading, Paragraph, Text, Button } from 'grommet'
-import { Rewind } from 'grommet-icons'
-import React from 'react'
-import { theme } from './theme'
-import { FullWidth } from './TwoCols'
-import { graphql, useStaticQuery, Link, navigate } from 'gatsby'
+import React, { useContext } from 'react'
+import { Box, Heading, Text, Image, ResponsiveContext } from 'grommet'
+import { Calendar } from 'grommet-icons'
+import { graphql, useStaticQuery, navigate } from 'gatsby'
+import styled from 'styled-components'
 import Time from './Time'
 
-const colors = theme.global.colors
+const StyledH3 = styled(Heading)`
+  font-size: 20px;
+  line-height: 30px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+`
 
 export default function() {
-  const data = useStaticQuery(graphql`
-    query {
-      graphcms {
-        meetups(
-          orderBy: time_DESC
-          where: {
-            id_in: [
-              "ck8vxvl1k1a700104ygm809n9"
-              "ck8vxuq6g1a6i0104fcr7s9qm"
-              "ck8vxvl1k1a710104juon23a5"
-              "ck8vxp38w1h4x01645txt71eb"
-              "ck8vxpthc1h5m0164m62lfqvp"
-              "ck8vxp38w1h4t0164eujq7shc"
-            ]
-          }
-        ) {
-          id
-          name
-          time
+  const breakpoint = useContext(ResponsiveContext)
 
-          keyImage {
-            url
-          }
-          highlightImage {
-            url
-          }
-        }
-      }
-    }
-  `)
+  // const data = useStaticQuery(graphql`
+  //   query {
+  //     graphcms {
+  //       meetups(
+  //         orderBy: time_DESC
+  //         where: {
+  //           id_in: [
+  //             "ck8vxvl1k1a700104ygm809n9"
+  //             "ck8vxuq6g1a6i0104fcr7s9qm"
+  //             "ck8vxvl1k1a710104juon23a5"
+  //             "ck8vxp38w1h4x01645txt71eb"
+  //             "ck8vxpthc1h5m0164m62lfqvp"
+  //             "ck8vxp38w1h4t0164eujq7shc"
+  //           ]
+  //         }
+  //       ) {
+  //         id
+  //         name
+  //         time
+
+  //         keyImage {
+  //           url
+  //         }
+  //         highlightImage {
+  //           url
+  //         }
+  //       }
+  //     }
+  //   }
+  // `)
+
+  // const meetups = data.graphcms.meetups
+  const meetups = []
 
-  const meetups = data.graphcms.meetups
   return (
-    <Box pad={{ vertical: 'medium' }} direction="row-responsive" wrap={true}>
+    <Box
+      direction="row-responsive"
+      wrap={true}
+      margin={{ horizontal: '-10px', top: 'medium', bottom: 'large' }}
```

Comment:
use gap if possible and don't try to hack grommet's grid system on your own.

Reply:
This is not a hack and gap is not possible with wrap per their own instructions. :)

Reply:
well, from the perspective of flexbox that's a hack (that worked well for a long time of responsive grids, I know) -> here's a solution with plain CSS grid and gaps & dummy content: https://github.com/cod1ng-earth/ce-website/commit/8d8e0723a17ca686b504f0d094fc304e0a3e0932#diff-ca4a299972b3611f7cdd7de2d42a7aa6R61